WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -17,6 +17,7 @@ class SupportTableViewController: UITableViewController {
     private var tableHandler: ImmuTableViewHandler?
     private let userDefaults = UserPersistentStoreFactory.instance()
     private let featureFlagStore: RemoteFeatureFlagStore
```

Comment:
@guarani since we are changing this class to no longer be dependent on FeatureFlag I think we can remove this property as well?  Since SupportConfiguration has its own default parameter for FeatureFlag there should be no impact

Reply:
Good call! I didn't see this, will update.